cmake/podioMacros.cmake

```diff
@@ -297,21 +193,10 @@ function(PODIO_ADD_ROOT_IO_DICT dict_name CORE_LIB HEADERS SELECTION_XML)
   # the core headers
   LIST(FILTER HEADERS EXCLUDE REGEX .*SIOBlock.h)
 
-  add_library(${dict_name} SHARED)
-  target_link_libraries(${dict_name} PUBLIC
-    ${CORE_LIB}
-    podio::podio
-    ROOT::Core
-    )
-  target_include_directories(${dict_name} PUBLIC
-    $<BUILD_INTERFACE:${ARG_OUTPUT_FOLDER}>
-    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+  ROOT_GENERATE_DICTIONARY(${dict_name} ${HEADERS} LINKDEF ${selectionfile}
```

Comment:
Do we need to bump the minimum version of ROOT for this? Practically we have mainly been running with 6.26 and 6.28 in any case for quite some time, so we might want to do that in any case.

Reply:
Good idea

Reply:
I have it successfully running on 6.24. So I think we can do without for the moment

Reply:
I just realize our README still claims 6.08, which has probably not been tested in a year or two, so maybe we should also use this opportunity to bring that up to speed.

Reply:
OK. Will put it in there.